docs/guardrails_ai/getting_started.ipynb

```diff
@@ -148,7 +148,7 @@
     "\n",
     "Next, we want to specify the quality criteria for the output to be valid and corrective actions to be taken if the output is invalid. We can do this by adding a `format` tag to each field in the output schema. Format tags can either be enforced by Guardrails, or they can only be suggetions to the LLM. You can see the list of validators enforced by Guardrails [here](../api_reference/validators.md). Additionally, you can create your own custom validators, see examples here [1](../examples/no_secrets_in_generated_text.ipynb), [2](../examples/recipe_generation.ipynb), [3](../examples/valid_chess_moves.ipynb).\n",
     "\n",
-    "As an example, for our use case we specify that the `affected_area` of `symptoms` should be one of the following: `['head', 'neck', 'chest']`. For this, we use the [`valid-choices` validator](https://shreyar.github.io/guardrails/validation/#guardrails.validators.ValidChoices).\n",
+    "As an example, for our use case we specify that the `affected_area` of `symptoms` should be one of the following: `['head', 'neck', 'chest']`. For this, we use the [`valid-choices` validator](https://guardrails-ai.github.io/guardrails/validation/#guardrails.validators.ValidChoices).\n",
```

Comment:
it's expected for this link to not work yet right?

Reply:
correct! We need to merge this up after we do the repo transfer. This link actually looks broken in general though